src/searchForm/overlay/Overlay.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef, useState } from 'react'
+import CSSTransition from 'react-transition-group/CSSTransition'
+
+export const enterTransitionDuration = 350
+export const exitTransitionDuration = 500
+export const reducedMotionTransitionDuration = 0
+
+const overlayTimeout = {
+  enter: enterTransitionDuration,
+  exit: exitTransitionDuration,
+}
+
+type OverlayProps = {
+  closeOnBlur: () => void
+  shouldDisplay: boolean
+  children: React.ReactNode
+  className: string
+}
+
+export const Overlay = ({ closeOnBlur, shouldDisplay, children, className }: OverlayProps) => {
+  const overlayRef = useRef<HTMLDivElement>(null)
+  const [reducedMotion, setReducedMotion] = useState(false)
+
+  useEffect(() => {
+    setReducedMotion(window.matchMedia('(prefers-reduced-motion)').matches)
+  }, [])
+
+  return (
+    <div
+      ref={overlayRef}
+      onBlur={evt => {
+        if (!overlayRef.current.contains(evt.relatedTarget as Node)) {
+          closeOnBlur()
+        }
+      }}
+      className={className}
+    >
+      <CSSTransition
+        in={shouldDisplay}
+        classNames="kirk-searchForm-overlay-transition"
```

Comment:
Is there a reason to use constants for transition duration and not the class names?

Reply:
Done 👍